virttest/utils_test/libvirt.py

```diff
@@ -1159,6 +1159,54 @@ def thread_func_migration(self, vm, desturi, options=None,
                 self.RET_MIGRATION = False
                 self.RET_LOCK.release()
 
+    def migrate_pre_setup(self, desturi, params=None, cleanup=False):
+        """
+        # Setup before migration,
+        # 1. To enable migration ports using iptables
+        # 2. Turn off SMT for power8 machine in remote machine to migrate
```

Comment:
1. After https://github.com/avocado-framework/avocado-vt/pull/774 merge, you can use is directly without a new function here to call it.
2. There's already a step to do it on local machine( [here](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/env_process.py#L571 ) ), and it have bug for x86, so 
I think you can fix it, please check 'setup_test.disable_smt()'